shardingsphere-jdbc/shardingsphere-jdbc-core/src/main/java/org/apache/shardingsphere/driver/executor/DriverJDBCExecutor.java

```diff
@@ -48,10 +48,13 @@
     
     private final JDBCLockEngine jdbcLockEngine;
     
-    public DriverJDBCExecutor(final MetaDataContexts metaDataContexts, final JDBCExecutor jdbcExecutor) {
+    private final String schemaName;
+    
+    public DriverJDBCExecutor(final MetaDataContexts metaDataContexts, final JDBCExecutor jdbcExecutor, final String schemaName) {
```

Comment:
I don't think it's a good idea to pass `schemaName` in Executor

Reply:
It may be a better way to pass in the schemaName during execution.